mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -286,6 +288,48 @@ func resourceMongoDBAtlasCluster() *schema.Resource {
 					},
 				},
 			},
+			"labels": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Set: func(v interface{}) int {
+					var buf bytes.Buffer
+					m := v.(map[string]interface{})
+					buf.WriteString(m["key"].(string))
+					buf.WriteString(m["value"].(string))
+					return hashcode.String(buf.String())
+				},
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"key": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"value": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"plugin": {
```

Comment:
Since it looks like we'll always include this let's put a comment before plugin to explain it, add in something like:
// Special key/value pair to contain cluster created/updated via Terraform Provider and version.  
// Only used as information for customer requested support and to count number of clusters created/updated using the Provider.